### PR TITLE
add school_id to SchoolWelcomeWizard

### PR DESCRIPTION
**Pull request by aldavidson**
### Context

[Trello card 808](https://trello.com/c/sDl3yzu3/808-scope-schoolwelcomewizards-by-school-id) - as part of the larger ticket [649 - Handle multiple schools](https://trello.com/c/lDAgx85e/649-handling-multiple-schools), we need to add school_id to `SchoolWelcomeWizard`s. Each user should have their own welcome wizard for each of the schools they're associated with.

### Changes proposed in this pull request

* Add school_id to `SchoolWelcomeWizard`
* Make sure that wizards get the correct school_id when created
* Add the school name to each page title in the wizard

### CODE REVIEW

File: app/controllers/school/base_controller.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def set_school
   end
 
   def require_completed_welcome_wizard!
-    unless @user.school_welcome_wizard&.complete? || params[:controller] == 'school/welcome_wizard'
+    unless @user.school_welcome_wizards.find_by_school_id(@school.id)&.complete? || params[:controller] == 'school/welcome_wizard'
```
Comment (benilovj) [inline app/controllers/school/base_controller.rb:19]: `welcome_wizard_for(school)`?

**Review by benilovj — APPROVED** [review]
Looks legit.

File: app/controllers/school/base_controller.rb (hunk 1 above)
Comment (aldavidson) [reply app/controllers/school/base_controller.rb]: sure. done.

**Review by tonyheadford — APPROVED** [review]
LGTM

**Pull request merged**